app/components/moderation-list-row/component.js

```diff
@@ -39,35 +39,45 @@ export default Component.extend({
 
     noActions: computed.not('submission.reviewActions.length'),
 
+    // latest action attempted on the preprint
     latestAction: computed('submission.reviewActions.[]', function() {
         return latestAction(this.get('submission.reviewActions'));
     }),
 
+    // first three contributors to the preprint
     firstContributors: computed('submission.node.contributors', function() {
         return this.get('submission.node.contributors').slice(0, 3);
     }),
 
+    // count of contributors in-addition to the first three
     additionalContributors: computed('submission.node.contributors', function() {
         return this.get('submission.node.contributors.content.meta.pagination.total') - 3;
     }),
 
-    gtDay: computed('submission.dateLastTransitioned', function() {
-        return moment().diff(this.get('submission.dateLastTransitioned'), 'days') > 1;
-    }),
-
-    relevantDate: computed('submission.dateLastTransitioned', 'gtDay', function() {
-        return this.get('gtDay') ?
+    // translations for moderator action label
+    reviewedOnLabel: computed('submission.{reviewsState,dateLastTransitioned}', 'noActions', 'latestActionCreator', function() {
+        const gtDay = moment().diff(this.get('submission.dateLastTransitioned'), 'days') > 1;
```

Comment:
Ok, now that `reviewedOnLabel` and `submittedOnLabel` are more parallel, it's easier to see what's duplicated and we can DRY it up. Maybe a function like this would help?
```js
function friendlyDate(rawDate) {
    const date = moment(rawDate),
          now = moment();
    const gtDayAgo = now.diff(date, 'days') > 1;
    const formattedDate = gtDayAgo ?
        date.format('MMMM DD, YYYY') 
        moment.min(date, now).fromNow();
    return [formattedDate, gtDayAgo];
}
```